### PR TITLE
JaCoCo maven plugin 0.8.5

### PR DESCRIPTION
**Pull request by sullis**

### CODE REVIEW

**Comment by rvema** [comment]
@sullis Can you sign the CLA if you have not already? link to CLA - http://hygieia.github.io/Hygieia/contribute.html

**Comment by sullis** [comment]
@rvema  done

File: pom.xml [hunk 1]
```diff
@@ -292,13 +292,13 @@
             <artifactId>commons-io</artifactId>
             <version>2.1</version>
         </dependency>
-        
+
```
Comment (rvema) [inline pom.xml]: please remove the whitespace/carriage return

File: pom.xml [hunk 2]
```diff
@@ -292,13 +292,13 @@
...
         <dependency>
             <groupId>javax.annotation</groupId>
             <artifactId>javax.annotation-api</artifactId>
             <version>1.3.2</version>
         </dependency>
-        
+
```
Comment (rvema) [inline pom.xml]: please remove the whitespace/carriage return

**Pull request merged**